### PR TITLE
Make script to import all data

### PR DESCRIPTION
**Pull request by athai**
## Changes in this PR.

- [X] Make print statements more descriptive
- [X] Create a script that runs all the other import script in the appropriate order
- [x] Add check to search for csvs in different directory (used for Ansible)

## Testing this PR.

1. Make sure no data is loaded in your version of the StreamWebs db
2. Do docker
3. ``python streamwebs_frontend/manage.py makemigrations``
4. ``python streamwebs_frontend/manage.py migrate``
5. ``cd data_scripts``, then ``./get_all.sh``

### Expected Output.

* Either log into the Django admin panel to view all data sheets or
* ``python manage.py dbshell`` and ``SELECT COUNT(*) FROM streamwebs_[model-name];``

Data should be loaded into:
* sites
* schools
* users
* macros
* water quality
* water quality samples
* riparian transects
* transect zones

```
[root@df54b09b79e4 data_scripts]# ./get_all.sh 
Sites loaded.
Schools loaded.
Users loaded.
Macroinvertebrates loaded.
Basic info loaded.
Water temperature loaded.
Air temperature loaded.
Dissolved oxygen loaded.
pH loaded.
Turbidity loaded.
Salinity loaded.
Conductivity loaded.
Total solids loaded.
BOD loaded.
Ammonia loaded.
Nitrite loaded.
Nitrate loaded.
Phophates loaded.
Fecal coliform loaded.
Measurement tools loaded.
Riparian Transects loaded.
Zone 1 loaded.
Zone 2 loaded.
Zone 3 loaded.
Zone 4 loaded.
Zone 5 loaded.

```

Output for ``SELECT COUNT(*) FROM streamwebs_site;``
```
streamwebs=# SELECT COUNT(*) FROM streamwebs_site;
 count 
-------
   242
(1 row)
```

### CODE REVIEW

File: data_scripts/get_additional_wq.py [hunk 1]
```diff
@@ -7,15 +7,30 @@
 from django.core.wsgi import get_wsgi_application
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "streamwebs_frontend.settings")
-proj_path = "/opt/streamwebs/streamwebs_frontend/"
+proj_path = "/home/centos/streamwebs/streamwebs_frontend/"
```
Comment (Kennric) [inline data_scripts/get_additional_wq.py]: This path is going o be different on the app servers - it needs to be relative or set from config.

Comment (athai) [inline data_scripts/get_additional_wq.py]: I changed the dockerfile/docker-compose to set the project up in ``/home/centos`` to match Openstack. 

How do I set the path from config?

**Comment by leian7** [comment]
+1

**Comment by Kennric** [comment]
+1

**Pull request merged**